esx_service/vmdk_ops.py

```diff
@@ -847,16 +848,31 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
     if not datastore:
         datastore_url = default_datastore_url
         datastore = default_datastore
+        use_default_ds = True
```

Comment:
Why? I think the "known datastore" should be all the datastores in the host, right?